core/src/luigi-config/customResources.js

```diff
@@ -123,3 +123,15 @@ export async function getCustomResources(authData) {
   }
   return [];
 }
+
+export async function getSchemas() {
```

Comment:
Can we rename it? For me, the name suggests that we're taking all the CRDs schemas, while we're taking our own extensibility schemas